R/plot.parameters_model.R

```diff
@@ -14,6 +14,15 @@
 #' @inheritParams plot.see_cluster_analysis
 #' @inheritParams plot.see_check_normality
 #' @inheritParams plot.see_parameters_brms_meta
+#' @param show_estimate Should the point estimate of each parameter be shown?
+#'   (default: `TRUE`)
+#' @param show_interval Should the compatibility interval(s) of each parameter
+#'   be shown? (default: `TRUE`)
+#' @param show_density Should the compatibility density (i.e., posterior,
+#'   bootstrap, or confidence density) of each parameter be shown?
+#'   (default: `FALSE`)
+#' @param log_scale Should exponentiated coefficients (e.g., odds-ratios) be
+#'   plotted on a log scale? (default: `FALSE`)
```

Comment:
Out of curiosity: Is defaulting to `FALSE` the best practice here?

Reply:
I think so. Previously we defaulted to a log scale, but in my experience, folks tend to have a much harder time interpreting log scales than natural ones. Open to other thoughts, but my thinking is that, if somone asked for odds ratios instead of log odds ratios, it's better to give them an exponential scaled plot

Reply:
Yes, I'd also say that `FALSE` is the better default here.